### DISCUSSION TITLE
custom template component added in 6.8.0

### DISCUSSION THREAD

**Discussion by tiphaineruy** [keycloakify/keycloakify#200] [Q&A]
Is there an example somewhere on how to use the new feature in 6.8.0 to inject a custom template ?
I guess something like that:
```
  <KcApp
    Template={() => <div>template</div>}
...
  </KcApp>
```

But this just overwrites everything

**Comment by garronej** [comment]
Hi @tiphaineruy,  
There is no exmple yet sorry but you can copy paste [the bultin template](https://github.com/InseeFrLab/keycloakify/blob/main/src/lib/components/Template.tsx) and start modifying it.  

If you want to start from something really basic this would be it:  

```tsx
import type { TemplateProps } from "keycloakify/lib/components/Template";

function MyTemplate(props: MyTemplateProps){
    const { formNode } = props;
    return <div>{formNode}</div>;
}

//...

  <KcApp
    Template={MyTemplate}
    //...
  </KcApp>

```

Good luck
